dockyard/api/app.py

```diff
@@ -0,0 +1,42 @@
+import pecan
+
+from dockyard.api import config as api_config
+from oslo_config import cfg
+from pecan import make_app
+
+from dockyard import model
+
+
+API_SERVICE_OPT = [
+    cfg.PortOpt('port',
+                 default=5869,
+                 help='Port for the dockyard service.'),
+    cfg.IPOpt('host',
+              default='127.0.0.1',
+              help='Listening address for dockyard service'),
+]
+
+CONF = cfg.CONF
```

Comment:
I am not sure but there must be a way to define values of these parameters through configuration file. So that these default values can be overridden. :